otlp/src/otlp/logs/component.cpp

```diff
@@ -51,19 +64,47 @@ LoggerComponent::LoggerComponent(const components::ComponentConfig& config,
   logger_config.attributes_mapping =
       config["attributes-mapping"]
           .As<std::unordered_map<std::string, std::string>>({});
+
+  if (config.HasMember("sinks")) {
+    auto sinks = config["sinks"];
+    auto logs_sink_str = sinks["logs"].As<std::string>("otlp");
+    logger_config.logs_sink = GetSinkTypeFromString(logs_sink_str);
+    auto tracing_sink_str = sinks["tracing"].As<std::string>("otlp");
```

Comment:
you may use 
Parse(const yaml_config::YamlConfig& value, formats::parse::To<Sink>)
instead

Reply:
So I will need to implement Parse function, I consider it redundant. Do you insist?

Reply:
there is SinkType, you may simply write Parse() instead of GetSinkTypeFromString()
I mean parser for Sink, not the whole subsection